docs/topics/typecasts.md

```diff
@@ -94,114 +94,3 @@ val x: String? = y as? String
 ```
 
 Note that despite the fact that the right-hand side of `as?` is a non-null type `String`, the result of the cast is nullable.
-
-## Type erasure and generic type checks
```

Comment:
Makes sense to give a link to the new location of the moved text, so that the users who come here by an anchored link don't feel themselves lost.

Reply:
Added a section with a link as we do in [some other pages](https://kotlinlang.org/docs/exceptions.html#java-interoperability). [Commit.](https://github.com/JetBrains/kotlin-web-site/pull/2409/commits/81bda1cf69b851bb954fa3ba6d3ac8fad552a06a)